infection.json.dist

```diff
@@ -0,0 +1,30 @@
+{
+    "source": {
+        "directories": [
+            "src/Psl/"
+        ],
+        "excludes": [
+            "RandomSequence"
+        ]
+    },
+    "timeout": 10,
+    "logs": {
+        "text": "var/infection.log",
+        "summary": "var/summary.log",
+        "json": "var/infection-log.json",
+        "perMutator": "var/per-mutator.md",
+        "github": true,
+        "badge": {
+            "branch": "2.0.x"
```

Comment:
You could use a regex here: `/^\d+\.\d+\.x$/` perhaps? 

See https://github.com/infection/site/pull/220/files